doc/vrrp/VRRP_Adaptation_HLD.md

```diff
@@ -458,7 +458,7 @@ admin@sonic:~$ redis-cli -n 1 hgetall "ASIC_STATE:SAI_OBJECT_TYPE_ROUTER_INTERFA
 
 ![VRRP ADD/DEL VRRP instance Flows](images/VRRP_Config_Instance_Flow.png)
 
-macvlanmgrd:
+vrrpmgrd:
   - Listens to VRRP create, delete and parameter change in CONFIG DB
   - Upon change
     - Add/del VRRP instance corresponding Macvlan device to kernel with IPs and state.
```

Comment:
[doc/vrrp/images/VRRP_Architecture_Diagram.png](https://github.com/sonic-net/SONiC/pull/1834/files#diff-9b3f93693b3201f4f82520d7ba8fa786d9c36792712124e16c5c2828395372a6)
Why do we have vrrpmgrd part of bgp container and vrrpsyncd part of swss container? sorry missed to comment on this earlier.
I thought, like other modules, we need to have vrrpmgrd part of swss and vrrpsyncd part of bgp container, agree?